themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -0,0 +1,9 @@
+@import '~terra-mixins/lib/Mixins';
+
+:global {
+    --terra-dev-site-blockquote-color: #24292e;
+    --terra-dev-site-content-background-color: #fff;
```

Comment:
This should be added to [src/wrappers/orion-fusion-theme/ContentLoaded.module.scss](https://github.com/cerner/terra-dev-site/blob/main/src/wrappers/orion-fusion-theme/MarkdownWrapper.module.scss)

Reply:
Removed it here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8